src/Data/Formatter/DateTime.purs

```diff
@@ -210,16 +225,26 @@ padQuadrupleDigit i
   | i < 1000 = "0" <> (show i)
   | otherwise = show i
 
+-- | Format a DateTime according to the format defined in the given `Formatter`
 format ∷ Formatter → DT.DateTime → String
 format f d = foldMap (formatCommand d) f
 
+-- | Format a DateTime according to the format defined in the given format string.
+-- | If the format string is empty, will return a `Left` value. Note that any 
```

Comment:
I could use a spot check on that statement.  From what I could tell, the only situation that would return a `Left` value was if the string is empty (from `List.some` on 164), but literally anything else would parse as a `Formatter` and just have `Placeholder` for any unrecognized portions of the input string.

Reply:
hmmm, I ran this:
```PureScript
quickCheck' 100_000 \str -> 
  (if str == "" then true else isRight $ parseFormatString str)
  <?> "Test failed for input '" <> str <> "'"
```
and got
> Test failed for input 'ᆌH졶簮'

So any advice what this message should say?